sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/VMTestBase.cs

```diff
@@ -888,6 +888,41 @@ protected string CreateProximityPlacementGroup(string rgName, string ppgName)
             return vm;
         }
 
+        protected DedicatedHostGroup CreateDedicatedHostGroup( string rgName, string dedicatedHostGroupName)
+        {
+            m_ResourcesClient.ResourceGroups.CreateOrUpdate(
+                   rgName,
+                   new ResourceGroup
+                   {
+                       Location = m_location,
+                       Tags = new Dictionary<string, string>() { { rgName, DateTime.UtcNow.ToString("u") } }
+                   });
+
+            DedicatedHostGroup dedicatedHostGroup = new DedicatedHostGroup()
+            {
+                Location = m_location,
```

Comment:
This won't work anymore as we have made platformFaultDomainCount mandatory

Reply:
It is optional in Swagger spec.